gen/conflictgen.py

```diff
@@ -85,25 +85,38 @@ def extend_ground_position(cls, initial: Point, max_distance: int, heading: int,
         """Finds the first intersection with an exclusion zone in one heading from an initial point up to max_distance"""
         extended = initial.point_from_heading(heading, max_distance)
         if theater.landmap is None:
-            # TODO: Why is this possible?
```

Comment:
This used to happen when Nevada had a typo in the landmap name.  It should only occur now if a new map were added that we don't have a landmap for.  @DanAlbert